src/path_generation/.gitignore

```
@@ -0,0 +1 @@
+version.py
```

```suggestion
/version.py
```
Adding a leading `/` will force `git` to only consider the file in the directory vs all files named `version.py` in sub-directories with respect to the `.gitignore`.